README.md

```diff
@@ -14,6 +14,21 @@ You can find us on Twitter [@MozillaReality](https://twitter.com/mozillareality)
 git clone git@github.com:MozillaReality/FirefoxReality.git
 ```
 
+*Clone the third-party repo.*
+
+If you're developing for the Oculus, Snapdragon VR, or VIVE, you need to clone the repo with third-party SDK files. 
+
+```bash
+git clone git@github.com:MozillaReality/FirefoxReality-android-third-party.git third_party
+```
+
+This repo is only available to Mozilla employees. If you have access to the relevant SDK but not this repo, you can manually place them here:
+
+ - `third_party/ovr_mobile/` for Oculus (should contain a `VrApi` folder)
+ - `third_party/svr/` for Snapdragon (should contain a `libs` folder, among other things)
+ - `third_party/wavesdk/` for Vive (should contain a `build` folder, among other things)
```

Comment:
There was already instructions in the readme for wave. See the section `If you want to build FirefoxReality for WaveVR SDK`.